api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -484,7 +490,7 @@ def blow_out(
         if isinstance(target, validation.WellTarget):
             if target.well.parent.is_tiprack:
                 _log.warning(
-                    "Blow_out being performed on a tiprack. "
+                    "Blow_out being performed on a tip rack. "
```

Comment:
This isn't a docstring and could have an effect on tests. Revert.
```suggestion
                    "Blow_out being performed on a tiprack. "
```